tests/scripts/thread-cert/node_cli.py

```diff
@@ -192,6 +192,10 @@ def get_addr16(self):
         self.pexpect.expect('Done')
         return addr16
 
+    def get_router_id(self):
+        rloc16 = self.get_addr16()
```

Comment:
rloc16 here should be a string, you should convert it to integer before shifting. it might be the root cause for the travis failures.

another defect is that get_addr16() would be called each time `get_router_id()` , in test 5.3.6, there may be up to 24 calls on the same device. (Many 'rloc16' cli you could see on the travis building).  I think you could get the rloc16 of the device once, and define a command API to get the router id it represents. Thoughts?

Reply:
How about refining the `get_addr16()` to something like this:
```python
def get_addr16(use_cache=True):
```